File: benchmarks/runtime/foreach_iter.lua
local n = 100000
local i = 0
local sum = 0
while i < n do
  local arr = { i, i + 1, i + 2, i + 3, i + 4 }
  for _, item in ipairs(arr) do
    sum = sum + item
  end
  i = i + 1
end
if sum <= 0 then os.exit(1) end
print(sum)

File: benchmarks/runtime/foreach_iter.js
let n = 100000;
let i = 0;
let sum = 0;
while (i < n) {
  const arr = [i, i + 1, i + 2, i + 3, i + 4];
  for (const item of arr) {
    sum += item;
  }
  i += 1;
}
if (sum <= 0) process.exit(1);
console.log(sum);
